desktop/apps/auction2/components/page/Banner.js

```diff
@@ -62,7 +41,47 @@ export default function Banner (props) {
 }
 
 Banner.propTypes = {
+  BackgroundBanner: PropTypes.func.isRequired,
   auction: PropTypes.object.isRequired,
   isLiveOpen: PropTypes.bool.isRequired,
   liveAuctionUrl: PropTypes.string
 }
+
+const mapStateToProps = (state) => {
+  const {
+    auction,
+    isLiveOpen,
+    liveAuctionUrl
+  } = state.app
+
+  const { cover_image, name } = auction.toJSON()
+
+  const BackgroundBanner = ({ children }) => {
+    const imageUrl = get(cover_image, 'cropped.url', '')
+
+    return (
+      <div
+        className='auction2-banner-live-bg'
+        style={{ backgroundImage: `url('${imageUrl}')` }}
+        alt={name}
+      >
+        {children}
+      </div>
+    )
+  }
+
+  BackgroundBanner.propTypes = {
+    children: PropTypes.node
+  }
```

Comment:
I once took a grad class called Theory of Programming Languages, in which we painstakingly derived how literally every programming language feature can be accomplished with function composition and application 😄 

To be more to the point, one could say that there's a Redux norm that what goes on in the selectors is limited to reinterpretation of raw data. I've become a big fan of trying to stick to a narrow set of norms, until there's a big gain to be had from breaking them, and then that defines a new norm.

Here, it seems like we could get this same effect by making `BackgroundBanner` an independent component (even if still in the same file) that takes an `imageUrl` and `name` as props. No need for `Banner` to access it via props when it can simply see it in its lexical scope.